src/main.cpp

```diff
@@ -1324,12 +1324,27 @@ bool ReadBlockFromDisk(CBlock& block, const CBlockIndex* pindex)
 
 CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams)
 {
```

Comment:
Yes; it is used in `CreateNewBlock()` in the miner, and `ConnectBlock()` and `ContextualCheckBlock()` in `main.cpp`.